internal/command/deploy/machines_deploymachinesapp.go

```diff
@@ -408,7 +408,11 @@ func (md *machineDeployment) updateMachineByReplace(ctx context.Context, e *mach
 
 	newMachineRaw, err := md.flapsClient.Launch(ctx, *e.launchInput)
 	if err != nil {
-		return err
+		if strings.Contains(err.Error(), "could not reserve resource for machine") {
```

Comment:
Is the error coming back from flaps a generic string, or does it have a type? If it's just a generic error, switch it to a type so we can do proper error inspection with `errors.Is` or `errors.As`. Otherwise if the error message changes at all this code won't get hit anymore.